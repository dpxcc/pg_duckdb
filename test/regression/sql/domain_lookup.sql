create domain my_string as varchar;
CREATE TABLE t(a my_string);

insert into t values ('test');
insert into t values ('apple');
select a > 'hello' from t;

DROP TABLE t;
drop domain my_string;
