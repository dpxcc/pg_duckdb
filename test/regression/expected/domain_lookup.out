create domain my_string as varchar;
CREATE TABLE t(a my_string);
insert into t values ('test');
insert into t values ('apple');
select a > 'hello' from t;
 ?column? 
----------
 t
 f
(2 rows)

DROP TABLE t;
drop domain my_string;
